dask/compatibility.py

```diff
@@ -56,10 +60,26 @@ def _getargspec(func):
     range = xrange
     reduce = reduce
     operator_div = operator.div
+    FileNotFoundError = IOError
 
     def _getargspec(func):
         return inspect.getargspec(func)
 
+    def gzip_decompress(b):
+        f = gzip.GzipFile(fileobj=BytesIO(b))
+        result = f.read()
```

Comment:
I don't know that gzipfiles need to be closed at all, just presuming the facility exists for a reason, and so we should ensure it gets called; if all it does is call close on the underlying file object (bytesIO), then never mind.